tests/test_parsers.py

```diff
@@ -410,3 +410,26 @@ def test_round_trip(self):
 
         # This checks that nothing funny gets added unexpectedly
         self.assertEqual(expected_prefix_map, reconsitited_msdf.prefix_map)
+
+    def test_round_trip_tsv_json_tsv(self):
+        """Test TSV => JSON => TSV using convert() + parse()."""
+        input_tsv = test_data_dir.joinpath("basic_subset.tsv")
+        input_msdf = parse_sssom_table(input_tsv)
+        input_msdf.clean_prefix_map()
+        with TemporaryDirectory() as directory:
+            directory = Path(directory)
+            json_path = directory.joinpath("basic_subset.json")
+            tsv_path = directory.joinpath("json_to_tsv.tsv")
+            with json_path.open("w") as jfile:
+                convert_file(input_path=input_tsv, output_format="json", output=jfile)
+            with tsv_path.open("w") as tfile:
+                parse_file(
+                    input_path=str(json_path),
+                    input_format="json",
+                    prefix_map_mode="merged",
+                    output=tfile,
+                )
+            msdf = parse_sssom_table(tsv_path)
+            pd.testing.assert_frame_equal(input_msdf.df, msdf.df)
```

Comment:
cool. Never ever heard of that before! Nice!

Reply:
Thanks!